draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -295,6 +296,38 @@ For example:
 Content-Dictionary: :pZGm1Av0IEBKARczz7exkNYsZb8LzaMrV7J32a2fFG4=:
 ~~~
 
+# The 'compression-dictionary' Link Relation Type
+
+This specification defines the 'compression-dictionary' link relation type
+{{WEB-LINKING}} that provides a mechanism for a HTTP response to provide a URL
+for a compression dictionary that is related to, but not directly used by the
+current HTTP response.
+
+The 'compression-dictionary' link relation type indicates that fetching and
+caching the specified resource is likely to be beneficial as future requests
+are likely to support using the provided compression dictionary for encoding
+responses.
+
+Clients SHOULD fetch the provided resource at a time that they determine would
+be appropriate.
```

Comment:
In general, leave 2119 language for cases that have interoperability consequences.

```suggestion
Clients can fetch the provided resource at a time that they determine would
be appropriate.
```